Makefile

```diff
@@ -0,0 +1,15 @@
+SHELL		= /bin/bash
+
+.PHONY: all test
+all:		test
+
+# test -- build required WASM and execute unit tests
+test:
+	cargo build --features 'build_wasms' --manifest-path=crates/holochain/Cargo.toml
+	cargo test
```

Comment:
This is already a function in the nix-shell under `hc-merge-test` or `hc-test` I really don't think we need a third way to build holochain?

Reply:
No problem, removed.  The Makefiles I add are mostly documentary; I always have to figure out what incantations to run in order to build, test, etc., so I just encode them as a Makefile so I don't have to always figure it out from scratch each time I return to the project...